tests/test_data_type_inference.py

```diff
@@ -24,6 +24,19 @@ def test_inference():
     assert data_types.infer_data_type(
         pd.Series([None, None, np.nan, float("nan")]) == data_types.DataType.MISSING
     ), "Should be missing"
+    assert (
+        data_types.infer_data_type(pd.Series([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]))
```

Comment:
nitpick: I'd use `range` to be more concise: `pd.Series(range(10))`

Reply:
done

Reply:
probably forgot to push?

Reply:
indeed, fixed now